### ISSUE TITLE
Temporarily pause execution

### ISSUE THREAD

**Issue by brokyo** [httpete-ire/typer#18]
I'm using multiple instances of this directive and was hoping to trigger them one at a time through a combination of `start-trigger` and `on-typed` like this, roughly:

**View**
`<typer words="['foo', 'bar']" on-typed="vm.wordComplete()" start-trigger="vm.word1Active" repeat="true">should be changing </typer> and <typer words="['baz', 'foobarbaz']" on-typed="vm.wordComplete()" start-trigger="vm.word2Active">this too</typer>`

**Controller**
```
vm.wordComplete() {
    var randomNumber = Math.floor(Math.random()*2)
    if(randomNumber === 0){
     vm.word1Active = true;
     vm.word2Active = false;
    } else if(randomNumber === 1){
     vm.word1Active = false;
     vm.word2Active = true;
    }
}

```
but it seems `start-trigger` is exclusively for starting the text loop rather than triggering the writing of the following word. Am I missing some available function or is this just not how this is designed.

**Comment by httpete-ire** [comment]
@brokyo yeah you are correct in thinking that `start-trigger` only starts the text loop rather then go to the write the next word. 

Feel free to make a PR with the changes if you come up with a solution
